sdk/device_manager_test.go

```diff
@@ -811,15 +811,20 @@ func TestDeviceManager_AddDevice_aliasExists(t *testing.T) {
 
 func TestDeviceManager_AddDevice(t *testing.T) {
 	handler := DeviceHandler{Name: "foo"}
+	pid := &pluginID{uuid: uuid.NewSHA1(uuid.NameSpaceDNS, []byte("test"))}
 	m := deviceManager{
 		aliasCache:     NewAliasCache(),
 		tagCache:       NewTagCache(),
-		id:             &pluginID{uuid: uuid.NewSHA1(uuid.NameSpaceDNS, []byte("test"))},
+		id:             pid,
```

Comment:
pid is confusing. I think this is plugin id, but pid is normally process id.

Reply:
good call. didn't even think about that. should be updated now.